.github/workflows/build_image_commit.yml

```diff
@@ -0,0 +1,42 @@
+name: Build docker image commit
+on:
+  push:
+    branches:
+      - build-images
+  workflow_dispatch:
+    inputs:
+      commit_sha:
+        description: 'Commit SHA to build'
+        required: true
+
+
+jobs:
+  build-amd64-image_commit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.commit_sha || github.sha }}
+      - name: Get Short Commit Hash
+        id: short_commit
+        run: echo "::set-output name=SHORT_COMMIT::${GITHUB_SHA::8}"      
+      - name: Login to Github resgistry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64
+          push: true
+          tags: ghcr.io/simon-tl/axon:amd64-${{ steps.short_commit.outputs.SHORT_COMMIT  }}
```

Comment:
Wouldn't it be better to store the image at axonweb3?

Reply:
Because there are too many images to commit, and the images built by commit will not be used in the end, it is decided to put them in a temporary repo

Reply:
yes, the build push also fails if merged into axonweb3/axon repo commit